### PR TITLE
Soviet gear's buff, craftable scavenger uniform and armor.

### PR DESCRIPTION
**Pull request by Ecologistos**
## Description

Buffed commies' armor vests to be similar to those NCR uses, also added new undersuit and armor vest, both craftable and, _as a huge part of my additions_, based off that one game made by Ukrainians.

## Motivation and Context

A fella asked to buff armor, so I did it. Solved the problem of the vests being _somewhat_ obsolete, as you could always just use a potential opponent's one instead of yours.
Another fella asked to add this suit, so I did it. I also felt like giving a 'neutral' suit for wastelanders, so, yeah, have it.

## How Has This Been Tested?

Localhosted, fixed a ton of times.
## Screenshots (if appropriate):
I'm sleepy sorry.
## Changelog (necessary)
:cl:
add: Added scavenger gear.
tweak: Buffed foreign armor.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -46,14 +46,22 @@
 	desc = "The army flak jacket series PPS-3 provides a sufficient level of protection for leading war operation against hardly-equipped opponent."
 	icon_state = "army_armor"
 	item_state = "army_armor"
-	armor = list("melee" = 25, "bullet" = 30, "laser" = 25, "energy" = 30, "bomb" = 25, "bio" = 25, "rad" = 20, "fire" = 20, "acid" = 20)
+	armor = list("melee" = 45, "bullet" = 45, "laser" = 45, "energy" = 45, "bomb" = 40, "bio" = 40, "rad" = 35, "fire" = 35, "acid" = 35)
```
Comment (SwiGs1) [inline code/modules/clothing/suits/armor.dm:49]: Take these armour values down by 5 or more

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -46,14 +46,22 @@
...
 
 /obj/item/clothing/suit/armor/vest/sov_heavy
 	name = "foreign heavy armor vest"
 	desc = "This item comprises military PPS-5 series body armor with beryllium sputter modified for use in the Wasteland. It is ineffective in areas of high radioactive activity."
 	icon_state = "berill_5m"
 	item_state = "berill_5m"
-	armor = list("melee" = 40, "bullet" = 45, "laser" = 40, "energy" = 40, "bomb" = 35, "bio" = 40, "rad" = 35, "fire" = 35, "acid" = 20)
+	armor = list("melee" = 65, "bullet" = 65, "laser" = 65, "energy" = 65, "bomb" = 55, "bio" = 55, "rad" = 50, "fire" = 50, "acid" = 50)
```
Comment (SwiGs1) [inline code/modules/clothing/suits/armor.dm:56]: Take these down by 5-10

File: code/modules/clothing/suits/armor.dm (hunk 1 above)
Comment (Ecologistos) [reply code/modules/clothing/suits/armor.dm:49]: Alright, fair enough.

File: code/modules/clothing/suits/armor.dm (hunk 2 above)
Comment (Ecologistos) [reply code/modules/clothing/suits/armor.dm:56]: Of course.

**Pull request merged**